server/i18n-bootstrap/index.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import superagent from 'superagent';
+
+/**
+ * Internal dependencies
+ */
```

Comment:
The comment block doesn't really apply -- this isn't really a dependency :slightly_smiling_face: